### PR TITLE
Migrate meds to med.json part 2

### PR DESCRIPTION
**Pull request by Cyrano7**
Should nicotine gum have a addiction_potential value?

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
It's supposed to satisfy cravings while weaning you off, so it shouldn't really be prolonging the addiction.

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -380,8 +379,7 @@
             "type" : "consume_drug",
             "activation_message" : "You shoot up.",
             "tools_needed" : { "syringe" : 1 },
-            "effects" : [
-                {
+            "effects" : [ {
```
Comment (mugling) [inline data/json/items/comestibles/med.json]: You can go further still with this...

File: data/json/items/comestibles/med.json [hunk 2]
```diff
@@ -342,6 +377,40 @@
...
+        "addiction_potential" : 80,
+        "addiction_type" : "crack",
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You smoke your crack rocks.  Mother would be proud.",
+            "effects" : [ {
+                    "id" : "high",
+                    "duration" : 15
+                }
+            ]
```
Comment (mugling) [inline data/json/items/comestibles/med.json]: Have you tested this, that looks like a JSON error?

**Comment by Cyrano7** [comment]
@mugling like this?

File: data/json/items/comestibles/med.json [hunk 3]
```diff
@@ -322,11 +322,7 @@
         "use_action" : {
             "type" : "consume_drug",
             "activation_message" : "You take some codeine.",
-            "effects" : [ {
-                    "id" : "pkill2",
-                    "duration" : 180
-                }
-            ]
+            "effects" : [ { "id" : "pkill2", "duration" : 180 } ]
```
Comment (mugling) [inline data/json/items/comestibles/med.json]: Yes

File: data/json/items/comestibles/med.json [hunk 4]
```diff
@@ -550,15 +542,7 @@
...
-            "effects" : [ {
-                    "id" : "pkill3",
-                    "duration" : 80
-                },
-                {
-                    "id" : "pkill2",
-                    "duration" : 200
-                }
-            ]
+            "effects" : [ { "id" : "pkill3", "duration" : 80 }, { "id" : "pkill2", "duration" : 200 } ]
```
Comment (mugling) [inline data/json/items/comestibles/med.json]: No, split these similar to the other examples within the same file that have multiple effects

File: data/json/items/comestibles/med.json [hunk 5]
```diff
@@ -380,16 +542,8 @@
...
-                    "id" : "pkill3",
-                    "duration" : 80
-                },
-                {
-                    "id" : "pkill2",
-                    "duration" : 200
-                }
-            ]
+            "effects" : [ { "id" : "pkill3", "duration" : 80 },
+            { "id" : "pkill2", "duration" : 200 } ]
```
Comment (Coolthulhu) [inline data/json/items/comestibles/med.json]: It would look better with an indent here.

**Pull request merged**